### PR TITLE
Basic auth form with Google OAuth implementation

### PR DESCRIPTION
**Pull request by mjurkus**
Added screen where all authentication forms will be accessible.

- when skipping auth user will be authenticated as guest
- not sure about email prefilling from user's contacts? Do we need it at all? 

Design is something to be desired, but polishing could be done later.
[Screen 1](https://user-images.githubusercontent.com/427661/46574156-180d1780-c9a8-11e8-91c9-852dd92d235f.png)
[Screen 2](https://user-images.githubusercontent.com/427661/46574159-1e9b8f00-c9a8-11e8-86cf-7d9d2eb4f840.png)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -200,4 +200,11 @@
         <item>Susisiekti esant poreikiui</item>
     </string-array>
 
+    <string name="prompt_email">El. paštas</string>
+    <string name="prompt_password">Slaptažodis</string>
+    <string name="action_sign_in">Prisijungti</string>
+    <string name="action_skip_sign_in">Ne dabar</string>
+    <string name="permission_rationale">Contacts permissions are needed for providing email completions.</string>
```
Comment (vycius) [inline app/src/main/res/values/strings.xml]: Lithuanian translation is missing

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -14,13 +14,18 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <!-- To auto-complete the email text field in the login form with the user's emails -->
+    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
+    <uses-permission android:name="android.permission.READ_PROFILE" />
+    <uses-permission android:name="android.permission.READ_CONTACTS" />
```
Comment (vycius) [inline app/src/main/AndroidManifest.xml]: These permissions seem quite scary while installing the app.

**Review by vycius — APPROVED** [review]
Looks good. I am slightly concerned about newly introduced permissions

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (mjurkus) [reply app/src/main/AndroidManifest.xml]: Yeah. Lets remove this feature.

**Comment by mjurkus** [comment]
Removed email lookup functionality. From API 26 Android suggests autofill based on `autofillHints` either way.

**Review by vycius — APPROVED** [review]
Perfect

**Pull request merged**